core/templates/components/code-mirror/codemirror.component.ts

```diff
@@ -34,6 +34,9 @@ interface CodeMirrorMergeViewOptions {
 export class CodeMirrorComponent implements AfterViewInit, OnChanges {
   @Input() options!: CodeMirrorMergeViewOptions;
   // TODO(#13015): Remove use of unknown as a type.
+  // The property 'value' is dependent on other property 'localValue' from
```

Comment:
Please take care to make your comments grammatical and understandable:

The property 'value' is dependent on another property, 'localValue', from 'schema-based-editor'. Most components using localValue are currently in AngularJS, so its type cannot be determined for now.

Reply:
Done, will keep this mind!